dev/com.ibm.ws.jaxws.2.3.common/bnd.bnd

```diff
@@ -167,11 +166,6 @@ Service-Component: \
      implementation:=com.ibm.ws.jaxws.catalog.OASISCatalogApplicationBusListener; \
      provide:='com.ibm.ws.jaxws.bus.LibertyApplicationBusListener'; \
      properties:="service.vendor=IBM", \
-  com.ibm.ws.jaxws.http.transport.factory.provider; \
```

Comment:
This was a big source of the problems with jaxws-2.3. Because off the way that jaxws-2.2 used the integration layer to hook-up to the CXF HTTP Transport module, that same integration layer no longer works with jaxws-2.3 and CXF 3.3. So I've removed our Liberty specific extensions of the CXF transports http module, this includes this Service-Component.